inst/extdata/install-packages.R

```diff
@@ -0,0 +1,177 @@
+    } else {
+      displayProgress("Checking RTOOLS", success = FALSE, message = packageInstallationMessages$installRTools, suppressOutput = suppressOutput)
+      return()
+    }
+  }
+  if (Sys.which("make") == "") {
+    displayProgress("Checking RTOOLS", success = FALSE, message = packageInstallationMessages$RToolsNotFound, suppressOutput = suppressOutput)
+    return()
+  }
+
+  displayProgress("Installing rClr", suppressOutput = suppressOutput)
+  if (version$major == "4") {
+    install.packages(paste0("https://github.com/Open-Systems-Pharmacology/rClr/releases/download/v", rclrVersion, "/rClr_", rclrVersion, ".zip"), repos = NULL)
+  }
+  if (version$major == "3") {
+    install.packages(paste0("https://github.com/Open-Systems-Pharmacology/rClr/releases/download/v", rclrVersion, "-R3/rClr_", rclrVersion, ".zip"), repos = NULL)
+  }
+
+  packagePaths <- .releasePaths
+  if (developerVersion) {
+    packagePaths <- .developPaths
+  }
+
+  displayProgress("Installing ospsuite.utils", suppressOutput = suppressOutput)
+  install.packages(packagePaths$ospsuite.utils, repos = NULL)
+  displayProgress("Installing tlf", suppressOutput = suppressOutput)
+  install.packages(packagePaths$tlf, repos = NULL)
+  displayProgress("Installing ospsuite", suppressOutput = suppressOutput)
+  install.packages(packagePaths$ospsuite, repos = NULL)
+}
+
+#' Install osps packages and their dependencies into global library.
+#'
+#' @param updatePackages If `TRUE` (default), all installed packages will be
+#' updated prior to installation
+#' @param pkSimPath Path where PK-Sim is installed. If this is not specified
+#' (`NULL`), path is estimated by the `ospsuite` package.
+#' @param rtoolsPath Path to where rtools are installed. If `NULL` (default),
+#' the path is deduced from system environment variables.
+#' @param rclrVersion Version of rClr package. Default is 0.9.2 for Windows R4.
+#' @param developerVersion If `FALSE` (default), release verions of the packages
+#' will be installed. If `TRUE`, latest developer builds of the osps packages
+#' will be installed
+#' @param suppressOutput
+installPackagesGlobally <- function(updatePackages = TRUE, pkSimPath = NULL,
+                                    rtoolsPath = NULL, rclrVersion = "0.9.2",
+                                    suppressOutput = TRUE,
+                                    developerVersion = FALSE) {
+  installOSPPackages(
+    rtoolsPath = rtoolsPath, rclrVersion = rclrVersion,
+    suppressOutput = suppressOutput,
+    developerVersion = developerVersion
+  )
+
+  # Update all installed packages from CRAN
+  if (updatePackages) {
+    update.packages(ask = FALSE)
+  }
+
+  displayProgress("Testing PK-Sim connection", suppressOutput = suppressOutput)
+  flagConnection <- FALSE
+  try(flagConnection <- testPKSIMConnection(pkSimPath = pkSimPath))
+  if (!flagConnection) {
+    stop(message = packageInstallationMessages$PKSimLoadFails)
+  }
```

Comment:
`testPKSIMConnection `should be executed on Windows only

Reply:
Also in case of failed connection I would produce just a warning, not error.
Otherwise it looks for the user like the whole installation was failed - which is in fact not the case!

Reply:
Is now an optional task that can be triggered by the user.